### DISCUSSION TITLE
Just wanted to say this is amazing.

### DISCUSSION THREAD

**Discussion by ImitableLine** [martinet101/ElevenClock#557] [General]
From the features like hiding when clicked for 10 seconds, to change which monitors display the clock this is the perfect tool for this job.

Thanks, this has been great using this.

**Comment by marticliment** [comment]
Thank you for giving this kind of feedback. Without those comments and feature requests it would be impossible to develop those features, so also thank you all :)

**Comment by PescheHelfer** [comment]
I fully agree! You make our Windows 11 lives so much easier, thank you :)

**Comment by LeStahL** [comment]
Really nice work! Thanks for writing this. :) Really a fan of making Windows less annoying, and this tool sure helps!
